### DISCUSSION TITLE
A dynamic link library (DLL) initialization routine failed error

### DISCUSSION THREAD

**Discussion by farman1094** [vercel/next.js#58917] [App Router]
I am trying to create a nextjs project but when running `npm run dev` i am getting this error:

```bash
 ⚠ Attempted to load @next/swc-win32-ia32-msvc, but an error occurred: A dynamic link library (DLL) initialization routine failed.
\\?\D:\notion\notion\node_modules\@next\swc-win32-ia32-msvc\next-swc.win32-ia32-msvc.node
 ⨯ Failed to load SWC binary for win32/ia32, see more info here: https://nextjs.org/docs/messages/failed-loading-swc
 ```
 
 I have installed the C++ binaries that has been mentioned on the docs page but still getting this error. I have done everything that is mentioned on the page but the issue still exist. I am using window 11 pc. If there is a way to resolve this kindly tell.
 
 I have added `.babelrc` as well but still getting the error. 
 ```
 {
    "presets": ["next/babel"]
}
```

**Comment by Samuel-MM** [comment]
Did u fix it?

**Comment by akhaled01** [comment]
Hi there!

Ensure that the node installation matches your system's architecture

to check it, run `node -p "process.arch"` and check the output, it should match you system's architecture. If not, reinstall node with an installation that matches your system's Architecture

**Reply by niccdevs** [reply]
Thank you so much!

**Comment by sagnihotri773** [comment]
![image](https://github.com/vercel/next.js/assets/49466830/b3393f73-7c5f-4aba-9285-bfe384622d6c)

i need help to fix this issue

**Reply by Samuel-MM** [reply]
Did u run node -p "process.arch" to check node installation. At least it worked for me. If it has a different version, you'll need to reinstall it, as @akhaled01 said

**Reply by danieljcksn** [reply]
The node installation matches my pc's arch, and the problem still occurs :/
